pages/integrations/sso.md

```diff
@@ -73,6 +73,21 @@ You can configure the session duration to any timeout between 6 hours and 8,760
 
 <%= image "session_duration/configure_session_duration.png", width: 623, height: 315, alt: "Screenshot of the Buildkite SSO Session Duration Configuration" %>
 
+## SSO session IP address pinning
+
+> 📘
+> Session IP Address Pinning is an Enterprise only feature.
+
+You can configure the SSO Session IP Address Pinning to require re-authentication when user IP changes.
+
+To set the Session IP Address Pinning you can either use the [GraphQL API](/docs/apis/graphql/graphql-cookbook#update-the-sso-provider-session-ip-address-pinning) or complete the following steps via the settings interface.
```

Comment:
General nitpicks about capitalization etc + we should move describing the location in the UI to the steps of the procedure. I'll update the procedure once I see screenshots of the flow ([requested here](https://linear.app/buildkite/issue/FDN-2327#comment-cce8832a)).

```suggestion
To set up SSO session IP address pinning, use the [GraphQL API](/docs/apis/graphql/graphql-cookbook#update-the-sso-provider-session-ip-address-pinning) or complete the following steps in the Buildkite dashboard:
```